.circleci/docker/common/install_cudnn8.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
+mkdir tmp_cudnn && cd tmp_cudnn
+wget -q https://developer.download.nvidia.com/compute/redist/cudnn/v8.3.2/local_installers/11.5/cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz -O cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz
```

Comment:
Nit, please use `curl` which does not have 2Gb download limit and also can figure out the name of the file
```suggestion
curl -OLs https://developer.download.nvidia.com/compute/redist/cudnn/v8.3.2/local_installers/11.5/cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz
```